7.4.цикл_while/7.4.14.py

```diff
@@ -0,0 +1,13 @@
+witchprice = int(input())
```

Comment:
В Python по PEP8 используется snake_case, значит имена переменных должны быть в нижнем регистре и разделяться нижними подчеркиваниями:
```suggestion
witch_price = int(input())
```